packages/http/src/Adaptor.js

```diff
@@ -15,7 +15,16 @@ import { request as sendRequest, xmlParser } from './Utils';
  */
 
 /**
- * Execute a sequence of operations.
+ * State object
+ * @typedef {Object} HttpState
+ * @private
```

Comment:
Marking it private means that this typedef won't be rendered in the docs as a standalone thing